### PR TITLE
feat: add Cilium branding page

### PR DESCRIPTION
**Pull request by k8isdead**
This PR adds a brand page 

[Preview](https://deploy-preview-132--cilium.netlify.app/brand/)

TODO: 

- [x] Add Brand guide links
- [x] Add Logo zip links

### CODE REVIEW

**Review by vannyle — APPROVED** [review]

File: src/components/pages/brand/logos/logos.jsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  <section className="relative bg-white pt-16 md:pt-24 lg:pt-28">
+    <Container>
+      <div className="flex flex-col items-start space-y-10 border-b border-gray-3 pb-10 lg:flex-row lg:justify-between lg:space-y-0">
+        <div className="flex flex-col space-y-2.5">
+          <Heading tag="h2" size="md">
+            Cilium Logo
+          </Heading>
+          <p className="flat-breaks lg:flat-none xl:flat-breaks">
+            Logo is a crucial part of our brand and one of our most valuable assets. <br /> We must
+            ensure its proper usage.
```
Comment (xmulligan) [inline src/components/pages/brand/logos/logos.jsx:52]:
```suggestion
            Our logo is in the shape of a honeycomb to represent the Cilium hive. <br /> Please use use it correctly and help us grow community.
```

File: src/components/pages/brand/logos/logos.jsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        <img className="order-2 lg:order-1" src={icon} alt="Cilium icon" width={592} height={406} />
+        <div className="order-1 flex flex-col space-y-2.5 lg:order-2">
+          <Heading tag="h2" size="lg">
+            Cilium icon
+          </Heading>
+          <p className="text-lg leading-relaxed lg:max-w-[488px]">
+            Icon is an identifying mark or symbol that doesn&apos;t contain the business name, like
+            a drawing or image that represents the business. Icon should only be used in cases where
+            a company icon or avatar is required (traditionally constrained to a perfect square or
+            circle). In all other cases, use the logo.
```
Comment (xmulligan) [inline src/components/pages/brand/logos/logos.jsx:122]:
```suggestion
            Our icon is our identifying mark that doesn&apos;t contain the name. The icon should only be used in cases where
            it is required (traditionally constrained to a perfect square or
            circle). In all other cases, use the logo.
```

File: src/components/pages/brand/legal-box/legal-box.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        By using CNCF brand materials you agree to the
+        <a href="https://www.linuxfoundation.org/terms/" target="_blank" rel="noreferrer">
+          {' '}
+          Linux Foundation Terms of Use
+        </a>
+        ,{' '}
+        <a href="https://www.linuxfoundation.org/trademark-usage/" target="_blank" rel="noreferrer">
+          the Trademark Usage Guidelines
+        </a>
+        , these CNCF branding guidelines, and{' '}
```
Comment (xmulligan) [inline src/components/pages/brand/legal-box/legal-box.jsx:18]:
```suggestion
        , <a href="https://www.linuxfoundation.org/terms/" target="_blank" rel="noreferrer">
          {' '}
          the CNCF branding guidelines
        </a>, and{' '}
```

Comment (xmulligan) [inline src/components/pages/brand/legal-box/legal-box.jsx:18]:
```suggestion
        , the CNCF branding guidelines, and{' '}
```

File: src/components/pages/brand/hero/hero.jsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+import Button from 'components/shared/button';
+import Container from 'components/shared/container';
+import Heading from 'components/shared/heading';
+
+import backgroundSmSvg from './images/background-sm.svg';
+import backgroundSvg from './images/background.svg';
+
+const title = 'Cilium Brand Guidelines';
+const description =
+  'We’ve created some guidelines to help you use our brand  and assets, including our logo, content and trademarks';
```
Comment (xmulligan) [inline src/components/pages/brand/hero/hero.jsx:12]:
```suggestion
  'Do you want to help grow the buzz around Cilium? We’ve created some guidelines to help you use our brand and assets, including our logo, content, and trademarks';
```

File: src/components/pages/brand/legal-box/legal-box.jsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        <a href="https://www.cncf.io/brand-guidelines/#legal" target="_blank" rel="noreferrer">
+          all CNCF rules and policies
+        </a>{' '}
+        as may be updated from time to time. You also acknowledge that CNCF is the sole owner of
```
Comment (xmulligan) [inline src/components/pages/brand/legal-box/legal-box.jsx:22]:
```suggestion
        which may be updated from time to time. You also acknowledge that CNCF is the sole owner of
```

**Comment by k8isdead** [comment]
@xmulligan I've implemented all your suggestions 👍🏻

File: src/components/pages/brand/logos/logos.jsx [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+  <section className="relative bg-white pt-16 md:pt-24 lg:pt-28">
+    <Container>
+      <div className="flex flex-col items-start space-y-10 border-b border-gray-3 pb-10 lg:flex-row lg:justify-between lg:space-y-0">
+        <div className="flex flex-col space-y-2.5">
+          <Heading tag="h2" size="md">
+            Cilium Logo
+          </Heading>
+          <p className="flat-breaks lg:flat-none xl:flat-breaks">
+            Our logo is in the shape of a honeycomb to represent the Cilium hive. <br /> Please use
+            use it correctly and help us grow community.
```
Comment (xmulligan) [inline src/components/pages/brand/logos/logos.jsx:52]:
```suggestion
            use it correctly and help us grow the community.
```

**Review by xmulligan — APPROVED** [review]

**Pull request merged**